src/App.js

```diff
@@ -30,39 +29,91 @@ const App = () => {
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
+  const [player, setPlayer] = useState(PLAYER_1);
+  const [winner, setWinner] = useState(null);
 
   // Wave 2
   // You will need to create a method to change the square 
   //   When it is clicked on.
   //   Then pass it into the squares as a callback
 
-
+  const markBoard = (id) => {
+    squares.forEach((row)=>{
+      row.forEach(square=> {
+        if (square.id === id && square.value === ''){
+          if (player === PLAYER_1) {
+            setPlayer(PLAYER_2);
+            square.value = 'x'
```

Comment:
Consider setting the value to the value of `PLAYER_2` to help avoid introducing bugs. 

```suggestion
            square.value = PLAYER_2
```